phpcs.xml.dist

```diff
@@ -125,6 +125,10 @@
         <exclude-pattern>*/tests/*</exclude-pattern>
     </rule>
 
+    <rule ref="SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod">
+        <exclude-pattern>lib/Doctrine/ORM/Mapping/Reflection/ReflectionPropertiesGetter.php</exclude-pattern>
+    </rule>
```

Comment:
I'd suggest using `// phpcs:ignore SlevomatCodingStandard.Classes.UnusedPrivateElements.UnusedMethod` only on the method that's being ignored instead of excluding the entire file.